lib/iris/cube.py

```diff
@@ -2124,7 +2124,14 @@ def __repr__(self):
     def _repr_html_(self):
         from iris.experimental.representation import CubeRepresentation
         representer = CubeRepresentation(self)
-        return representer.repr_html()
+        # Catch exceptions in the html repr rather than passing them,
+        # and so fall back to the default cube repr.
+        try:
+            result = representer.repr_html()
+        except (IndexError, ValueError):
+            result = None
```

Comment:
Should we warn that something's gone wrong? It might be weird to not get an html repr of a cube unexpectedly (my preference is no - Iris is loud enough already).